v/vault/Dockerfiles/1.5.3-ubi8/README.md

```diff
@@ -7,3 +8,38 @@ Docker run command:
         Server mode:
         docker run --cap-add=IPC_LOCK -e 'VAULT_LOCAL_CONFIG={"backend": {"file": {"path": "/vault/file"}}, "default_lease_ttl": "168h", "max_lease_ttl": "720h"}' ppc64le/vault server
 
+# Deploying Vault on Openshift
+```
+$ oc new-project vault-test
+--> Now using project "vault-test" on server "https://api.sneha-ppc.cp.fyre.ibm.com:6443".
```

Comment:
Can you replace the server address with `https://api-<cluster-address>:6443` ?
As no one can access the address as you are not providing username & password.

Reply:
Updated.